tools/data-handler/src/command-handler.ts

```diff
@@ -848,6 +856,32 @@ export class Commands {
     return { statusCode: 200 };
   }
 
+  // Run migrations to bring project to target schema version
+  private async migrate(
+    toVersion?: number,
+    options?: { backup?: string; timeout?: number },
+  ): Promise<requestStatus> {
+    if (!this.commands) {
+      return { statusCode: 500, message: 'Commands not initialized' };
+    }
+
+    const backupDir = options?.backup ? options.backup : undefined;
```

Comment:
```suggestion
    const backupDir = options?.backup
```
Same thing. Then there would also no need to create the variable